test/uk/gov/hmrc/customsservicestatus/controllers/AdminCustomsServiceStatusControllerSpec.scala

```diff
@@ -63,4 +63,14 @@ class AdminCustomsServiceStatusControllerSpec extends BaseSpec {
       }
     }
   }
+
+  "getPlannedWork" should {
+    "return OK and call the PlannedWorkService" in {
+
+      when(mockAdminCustomsStatusService.getPlannedWork).thenReturn(Future[List[OutageData]](fakePlannedWorks))
+
+      val result = controller.getPlannedWork(FakeRequest())
+      status(result) shouldBe OK
```

Comment:
Might be good to just add in a basic check that the content of the header of page is present.